chain/network/src/routing/graph/mod.rs

```diff
@@ -220,7 +224,15 @@ impl Inner {
         for peer in next_hops.keys() {
             self.peer_reachable_at.insert(peer.clone(), now);
         }
-        self.prune_unreachable_peers(now - self.config.prune_unreachable_peers_after);
+        // Prune unreachable peers from time to time.
+        if self
+            .last_time_peers_pruned
+            .map_or(true, |t| t < now - self.config.prune_unreachable_peers_after / 2)
```

Comment:
IIUC, the pruning runs every X/2 seconds and does pruning for the last X seconds.

Please add a comment about this `/2` pruning frequency.

Reply:
added.